Package.swift

```diff
@@ -1,4 +1,6 @@
-// swift-tools-version:5.3
+// swift-tools-version:5.4
```

Comment:
I think you also need to update the template `Package.swift` in RakutenAnalyticsSDK folder in the main repo

Reply:
@imaginaris You are right. Done.